src/posts/2020-11-23-getting-started-with-voiceover-ios.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title: Getting started with VoiceOver on iOS
+description: How-to guide to the basics of navigating around on an iPhone using Apple’s VoiceOver screen reader, included in iOS.
+category: Assistive Technology
+author: Anh Pham
+date: 2020-11-23
+further_reading:
+  - title: VoiceOver commands and gestures
+    url: https://support.apple.com/en-ca/guide/iphone/iph3e2e2281/ios
+    source: Apple
+  - title: Practice VoiceOver on iPhone Guide
+    url: https://support.apple.com/en-ca/guide/iphone/iph3e2e415f/ios
+    source: Apple
+eleventyNavigation:
+  key: {{ category }}
+  title: {{ title }}
+tags:
+  - assistivetechnology
+---
+
+## What is VoiceOver?
+
+VoiceOver is a screen-reading tool built into Apple Inc.'s iOS. It helps people navigate and use their iPhone without needing to see the screen. First introduced in 2009 with the iPod Shuffle and iPhone 3GS, VoiceOver has been in the picture in every Apple's new release ever since. 
+
+For developers, it is always a good idea to test out VoiceOver with your project on your chosen testing device.
+
+## How does it work?
+
+On an iPhone, VoiceOver responds to gestures (single tap, double tap, et cetera) and audibly communicates to us all the information we need to know (for example, what we have selected or done, the hints, values and traits of the items).
+
+## How to start using VoiceOver on iOS
+
+The easiest way to turn on VoiceOver on your iPhone is to:
+- Open Settings app on your iPhone.
+- Select Accessibility.
+- Under Vision, select VoiceOver.
+- Turn on VoiceOver.
```

Comment:
```suggestion

1. Open Settings app on your iPhone.
1. Select Accessibility.
1. Under Vision, select VoiceOver.
1. Turn on VoiceOver.
```

Can we make this an ordered list? I'm thinking since it's a set of ordered instructions the list type should reflect that.